roles/kubernetes/preinstall/tasks/0040-set_facts.yml

```diff
@@ -70,7 +70,7 @@
 
     - name: get currently configured nameservers
       set_fact:
-        configured_nameservers: "{{ resolvconf_slurp.content | b64decode | regex_findall('\\s*nameserver\\s*(.*)') | ipaddr }}"
+        configured_nameservers: "{{ resolvconf_slurp.content | b64decode | regex_findall('^nameserver\\s*(.*)', multiline=True) | ipaddr }}"
```

Comment:
Wondering if something like `^\\s*` might make more sense to allow indentation

Reply:
HI @floryut 

I test the resolve file with an indentation in ubuntu:
```
  nameserver 192.168.1.29
```
It does not work.
So these should not be indentation before the `nameserver`.

Reply:
Thank you for testing this 👍